src/components/deposit/Pay.js

```diff
@@ -14,84 +16,46 @@ function Pay(props) {
-  }
-
-  copyAddress = (evt) => {
-    this.hiddenCopyField.select()
-    document.execCommand('copy')
-    this.hiddenCopyField.blur()
-    this.setState({ copied: true })
-  }
-
-  render() {
-    const { btcAddress, lotInSatoshis, signerFeeInSatoshis, error } = this.props
-    const lotInBtc = (new BigNumber(lotInSatoshis.toString())).div(BitcoinHelpers.satoshisPerBtc.toString())
-    const signerFeeInBtc = (new BigNumber(signerFeeInSatoshis.toString())).div(BitcoinHelpers.satoshisPerBtc.toString())
-
-    const { copied } = this.state
+const PayComponent = ({ btcAddress, lotInSatoshis, signerFeeInSatoshis, error }) => {
+  const lotInBtc = (new BigNumber(lotInSatoshis.toString())).div(BitcoinHelpers.satoshisPerBtc.toString())
+  const signerFeeInBtc = (new BigNumber(signerFeeInSatoshis.toString())).div(BitcoinHelpers.satoshisPerBtc.toString())
 
-    const btcAmount = lotInBtc.toString()
-    const signerFee = signerFeeInBtc.toString()
-    const btcURL =
-      `bitcoin:${btcAddress}?amount=${btcAmount}&label=Single-Use+tBTC+Deposit+Wallet`
+  const btcAmount = lotInBtc.toString()
+  const signerFee = signerFeeInBtc.toString()
+  const btcURL =
+    `bitcoin:${btcAddress}?amount=${btcAmount}&label=Single-Use+tBTC+Deposit+Wallet`
 
-    return (
-      <div className="pay">
-        <div className="page-top">
-          <div className="qr-code">
-            <QRCode
-              value={btcURL}
-              renderAs="svg"
-              size={225} />
-          </div>
+  return (
+    <div className="pay">
+      <div className="page-top">
+        <StatusIndicator />
+      </div>
+      <div className="page-body">
+        <div className="step">
+          Step 2/5
         </div>
-        <div className="page-body">
-          <div className="step">
-            Step 2/5
-          </div>
-          <div className="title">
-            Pay: {btcAmount} BTC
-          </div>
-          <hr />
-          <div className="description">
-            <div>
-              Scan the QR code or click to copy the address below into your wallet.
-            </div>
-            <div className="custodial-fee">
-              <span className="custodial-fee-label">Signer Fee: </span>
-              {signerFee} BTC*
-            </div>
-          </div>
-          <div className="copy-address">
-            <div className="address" onClick={this.copyAddress}>
-              {btcAddress}
-            </div>
-            {
-              copied
-              ? <div className="copied">Copied!</div>
-              : ''
-            }
+        <div className="title">
+          Pay: {btcAmount} BTC
+        </div>
+        <hr />
+        <div className="description">
+          <div>
+            Scan the QR code or click to copy the address below into your wallet.
           </div>
-          <div className="error">
-            { error }
+          <div className="custodial-fee">
+            <span className="custodial-fee-label">Signer Fee: </span>
```

Comment:
At some point we should align the terminology here (i.e., use `signer-fee` as the class name as well).

Reply:
Addressed in 9e63614